### PR TITLE
Add onEnrollmentEntityChange

### PR DESCRIPTION
**Pull request by ciss1995**

### CODE REVIEW

File: src/enrollments/EnrollmentCollectionEntity.js [hunk 1]
```diff
@@ -44,4 +44,8 @@ export class EnrollmentCollectionEntity extends Entity {
 	getSearchEnrollmentsActions() {
 		return this._entity.getActionByName(Actions.enrollments.searchMyEnrollments);
 	}
+
+	onEnrollmentEntityChange(url, OnChange) {
```
Comment (ctwomey1) [inline src/enrollments/EnrollmentCollectionEntity.js]: Would you not want to know the change for all of the subentities?  I'm not sure i understand the use case

Comment (ctwomey1) [reply src/enrollments/EnrollmentCollectionEntity.js]: Additionally, this seems like it's registring for an onchange that may not be on the enrollmentEntityCollection.  which i'm not sure we want to do.

If you need to watch an individual enrollment Entity you might be better suited watching it as a full entity

Comment (ctwomey1) [reply src/enrollments/EnrollmentCollectionEntity.js]: i'd still like to understand this @ciss1995 despite it getting merged

Comment (m6jones) [reply src/enrollments/EnrollmentCollectionEntity.js]: I'd like to know too. Totally missed this but we shouldn't be adding the `url` parameter. It shouldn't require it.

**Pull request merged**